### PR TITLE
refactor seturl

### LINKED ISSUE #1244: Refactor setURL()
refactor `BenefitsDirectory.setURL()` to be similar to `Guided.setURL()`

### PR DESCRIPTION
**Pull request by sastels**
resolves #1244

### CODE REVIEW

File: pages/benefits-directory.js [hunk 1]
```diff
@@ -80,28 +77,30 @@ const mapDispatchToProps = dispatch => {
...
-    patronType: reduxState.patronType,
+    profileFilters: getProfileFilters(reduxState, props),
+    // patronType: reduxState.patronType,
     searchString: reduxState.searchString,
-    serviceType: reduxState.serviceType,
-    statusAndVitals: reduxState.statusAndVitals,
-    serviceHealthIssue: reduxState.serviceHealthIssue,
+    // serviceType: reduxState.serviceType,
+    // statusAndVitals: reduxState.statusAndVitals,
+    // serviceHealthIssue: reduxState.serviceHealthIssue,
```
Comment (smcmurtry) [inline pages/benefits-directory.js]: 🔥

File: pages/benefits-directory.js [hunk 2]
```diff
@@ -80,28 +77,30 @@ const mapDispatchToProps = dispatch => {
...
   t: PropTypes.func.isRequired,
   sortBy: PropTypes.string.isRequired,
   store: PropTypes.object,
-  patronType: PropTypes.string.isRequired,
+  // patronType: PropTypes.string.isRequired,
   searchString: PropTypes.string.isRequired,
-  serviceType: PropTypes.string.isRequired,
-  statusAndVitals: PropTypes.string.isRequired,
+  // serviceType: PropTypes.string.isRequired,
+  // statusAndVitals: PropTypes.string.isRequired,
```
Comment (smcmurtry) [inline pages/benefits-directory.js]: 🔥

**Review by smcmurtry — APPROVED** [review]
LGTM 🦑

**Comment by sastels** [comment]
[User answers as little profile questions as possible](https://trello.com/c/MF0pMzGT/20-user-answers-as-little-profile-questions-as-possible)

**Pull request merged**